devcareer/code-jammers-backend
# #174626687 Build Country and Culture Models [WIP]

### What does this PR do?
This PR designs the sequelize models with migrations and seeders for country and
culture

## Description of tasks to be completed?
- create country model
- create culture model
- create country migrations
- create culture migrations
- create country seeders
- create culture seeders

## How should this be manually tested/checked?
- Clone the PR
- Cd into the branch cd ft-countries-culture-models-174626687
-  Run npm install
- Create a .env file and define your database as follows:
    DEV_DATABASE_URL = `<development database url>`
    TEST_DATABASE_URL = `<test database url>`
    DATABASE_URL = `<production database url>`
-  Run this command  `npm run test` to check that all test are passing
-  Run this command `npm run migrate` to run migrations into your Database
-  Run this command `npm run migrate:undo` to undo migrations from Database
-  Run this command `npm run seed` to  insert some data into country and culture tables by default into Database
-  Run this command `npm run seed:undo` to undo seed from Database

What are the relevant pivotal tracker stories?
[#174626687](https://www.pivotaltracker.com/story/show/174626687)
